persistence/ravendb/outbox_disable-cleanup_raven_[,3].partial.md

```diff
@@ -0,0 +1,5 @@
+The maximum value allowed for both properties is `int.MaxValue * 2` milliseconds, which is between 49 and 50 days.
+
+The cleanup task can be disabled by specifying a value of `Timeout.InfiniteTimeSpan` for `SetFrequencyToRunDeduplicationDataCleanup`. This can be useful when an endpoint is scaled out and instances are competing to run the cleanup task.
```

Comment:
The current behavior is that cleanup still runs once after one 1 minute after the endpoint instance started no matter the configure frequency.

Reply:
This is covered in the note right after this.